src/test/resources/features/comments.feature

```diff
@@ -0,0 +1,28 @@
+Feature: comments
+
+  Background:
+    Given that I enter the conduit site
+    And I click on the Sign In link
+    And Authenticate me on the site
+
+  @HappyPath
+  Scenario: A comment is made successfully
+    When I click on the Global Feed link
+    And the system displays the list of items
+    And click on an article title in the list
+    And the system opens the article information
+    And write a comment to the article
+    And I click post comment
+    Then the new comment is added to the article
+    And the author and the date of the comment are displayed
+    And the comment is removed
+
+  @HappyPath
```

Comment:
I think this scenario is not part of the happy path 🤔 What do you think?

Reply:
it is changed by `@SadPath`